### PR TITLE
602/Add header back

### PR DESCRIPTION
**Pull request by ManuelMoeri**

### CODE REVIEW

**Review by MarcEgliP — APPROVED** [review]

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -9,13 +9,37 @@
...
+          %div.d-flex
+            %div.mb-1.mt-1
+              %img{:src=> "/assets/logo.svg",:height=>"32"}
+            %text.d-flex.align-items-end.ms-2.small-text= "4.4.0"
+          %ul.navbar.grey-color
+            %li.align-center.nav-pad.border-r-l.h-100
+              %pzsh-icon.scale-icon-06(name="user")
+              %text= "Curtis Jackson"
+            %li.align-center.nav-pad
+              %a.align-center{:href => "https://google.com"}
```
Comment (Robin481) [inline app/views/layouts/application.html.haml:25]: Can already link to the skills github issues here.

File: app/assets/stylesheets/styles.scss [hunk 2]
```diff
@@ -2,4 +2,65 @@
...
+  max-width: 60%;
+  flex-grow: 1;
+}
+
+.navbar {
+  flex-direction: row;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
```
Comment (Robin481) [inline app/assets/stylesheets/styles.scss:17]: General comment on the css:
Please do some nesting. 
`.scale-icon-08` for example doesn't need to exist in a global context. Just put it into the `navbar` class so it exists only in that context. 
Goes for the rest of the CSS classes as well. 
We probably don't want a global `nav-link` class that is not relevant in any other cases.

File: app/assets/stylesheets/styles.scss [hunk 3]
```diff
@@ -2,4 +2,65 @@
...
+.top-bar-color {
+  background-color: #3b7bbe
+}
+
+.grey-color {
+  color: #8e8e8e;
+}
+
+.small-text {
+  font-size: 10px;
```
Comment (Robin481) [inline app/assets/stylesheets/styles.scss:65]: There exist Bootstrap classes for changing the size of text.

File: app/assets/stylesheets/styles.scss [hunk 4]
```diff
@@ -2,4 +2,65 @@
...
+  transform: scale(0.8);
+}
+
+.border-r-l {
+  border-right: 1px solid;
+  border-left: 1px solid;
+}
+
+.top-bar-color {
+  background-color: #3b7bbe
```
Comment (Robin481) [inline app/assets/stylesheets/styles.scss:57]: Aren't there variables available from the puzzle shell for this?
If not define the color globally as a variable so we can use it in multiple places. Same goes for the grey below.

File: app/assets/stylesheets/styles.scss [hunk 5]
```diff
@@ -2,4 +2,65 @@
...
+  transform: scale(0.6);
+}
+
+.scale-icon-08 {
+  transform: scale(0.8);
+}
+
+.border-r-l {
+  border-right: 1px solid;
+  border-left: 1px solid;
```
Comment (Robin481) [inline app/assets/stylesheets/styles.scss:53]: Bootstrap borders? https://getbootstrap.com/docs/4.0/utilities/borders/#border-color

File: app/assets/stylesheets/styles.scss [hunk 6]
```diff
@@ -2,4 +2,65 @@
...
+}
+
+.align-center {
+  display: flex;
+  align-items: center;
+}
+
+.sticky-header {
+  position: sticky;
+  top: 0;
```
Comment (Robin481) [inline app/assets/stylesheets/styles.scss:40]: https://getbootstrap.com/docs/5.0/helpers/position/#fixed-top ?

File: app/assets/stylesheets/styles.scss [hunk 7]
```diff
@@ -2,4 +2,65 @@
...
+.nav-pad {
+  padding-right: 0.5rem;
+  padding-left: 0.5rem;
+}
+
+.nav-pad :hover {
+  cursor: pointer;
+}
+
+.align-center {
```
Comment (Robin481) [inline app/assets/stylesheets/styles.scss:33]: Bootstrap class?

File: app/assets/stylesheets/styles.scss [hunk 8]
```diff
@@ -2,4 +2,65 @@
...
+  text-decoration: none;
+}
+
+.nav-pad {
+  padding-right: 0.5rem;
+  padding-left: 0.5rem;
+}
+
+.nav-pad :hover {
+  cursor: pointer;
```
Comment (Robin481) [inline app/assets/stylesheets/styles.scss:30]: https://getbootstrap.com/docs/5.0/utilities/interactions/#pointer-events

File: app/assets/stylesheets/styles.scss [hunk 9]
```diff
@@ -2,4 +2,65 @@
...
+}
+
+.nav-link {
+  color: white;
+  text-decoration: none;
+}
+
+.nav-pad {
+  padding-right: 0.5rem;
+  padding-left: 0.5rem;
```
Comment (Robin481) [inline app/assets/stylesheets/styles.scss:26]: Bootstrap spacing?

File: app/views/layouts/application.html.haml [hunk 10]
```diff
@@ -9,13 +9,37 @@
...
+            %div.d-flex.justify-content-center
+              %ul.navbar
+                %li
+                  %a.nav-link.nav-pad= "Profil"
+                %li
+                  %a.nav-link.nav-pad= "Skill Suche"
+                %li
+                  %a.nav-link.nav-pad= "CV Suche"
+                %li
+                  %a.nav-link.nav-pad{href: "/skills"} Skillset
```
Comment (Robin481) [inline app/views/layouts/application.html.haml:40]: Use Rails routes please

**Review by Robin481 — CHANGES_REQUESTED** [review]

**Review by Robin481 — APPROVED** [review]

**Pull request merged**